grails-app/controllers/org/pih/warehouse/RoleInterceptor.groovy

```diff
@@ -63,6 +63,7 @@ class RoleInterceptor {
         'transactionEntry'          : ['edit', 'delete', 'save', 'update'],
         'user'                      : ['impersonate'],
         'productsConfigurationApi'  : ['downloadCategories', 'importCategories'],
+        'locationType'              : ['create', 'edit', 'delete'],
```

Comment:
how about the `update`, `save`?  I don't remember exactly how those rules work, but I know that `update` and `save` are usually different controller actions than `create` and `edit`